packages/itwin/grouping-mapping-widget/src/widget/components/GroupingMapping.scss

```diff
@@ -6,11 +6,8 @@
 
 // Widgets by default don't get horizontal scrollbars...
 /* stylelint-disable-next-line selector-class-pattern */
```

Comment:
You removed the styles for `.nz-widget-contentRenderer`, but didn't remove the comment above it. I think this may confuse other developers that come across this.

Reply:
I noticed it too late. Going in the next PR.